### PR TITLE
fix: in KMIP-Get operation, KeyFormatType must be editable

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: src/kms.ts [hunk 1]
```diff
@@ -150,15 +151,27 @@ export class KmsClient {
   /**
    * Retrieve a KMIP Object from the KMS
    * @param uniqueIdentifier the unique identifier of the object
-   * @param keyWrappingSpecification specifies keys and other information for wrapping the returned object
+   * @param options Additional optional options
```
Comment (ccorsin) [inline src/kms.ts:154]: Additional options is enough ? :)

File: src/kms.ts [hunk 2]
```diff
@@ -1254,14 +1272,14 @@ export class KmsClient {
   private async manageAccess(
     uniqueIdentifier: string,
     userIdentifier: string,
-    operationType: KMIPOperations,
+    operationTypes: KMIPOperations[],
```
Comment (ccorsin) [inline src/kms.ts:1275]: yay - a gift for Thibaud

File: src/kms.ts [hunk 3]
```diff
@@ -1190,10 +1208,10 @@ export class KmsClient {
       WrappingMethod.Encrypt,
       new EncryptionKeyInformation(encryptionKeyUniqueIdentifier),
     )
-    const object = await this.getObject(
-      uniqueIdentifier,
+    const object = await this.getObject(uniqueIdentifier, {
+      unwrap: true,
```
Comment (ccorsin) [inline src/kms.ts:1212]: is it OK/logic to set unwrap to true here, in a function named getWrappedKey ? If so, maybe rename this function ?

Comment (Manuthor) [reply src/kms.ts:1212]: true, removing it, thank you

File: src/kms.ts (hunk 1 above)
Comment (Manuthor) [reply src/kms.ts:154]: indeed!

**Pull request merged**